testing/nginx-naxsi/nginx.initd

```diff
@@ -1,42 +1,69 @@
 #!/sbin/openrc-run
 
-extra_started_commands="reload"
-extra_commands="configtest"
+description="Nginx http and reverse proxy server"
+extra_started_commands="reload reopen upgrade"
+
+cfgfile=${cfgfile:-/etc/nginx/nginx.conf}
+pidfile=/run/nginx/nginx.pid
+tmpdir=/var/lib/nginx/tmp
+command=/usr/sbin/nginx
+command_args="-c $cfgfile"
+required_files="$cfgfile"
 
 depend() {
-	need net
+#	need net => breaks LXC containers ##
```

Comment:
`need net` left alone - it does however stop `nginx` starting in `lxc` containers with ip addresses set in `container/config`